tests/baselines/reference/typeArgumentDefaultUsesConstraintOnCircularDefault.errors.txt

```diff
@@ -15,4 +16,12 @@ tests/cases/compiler/typeArgumentDefaultUsesConstraintOnCircularDefault.ts(5,19)
                       ~~~~~
 !!! error TS2322: Type '{}' is not assignable to type 'string'.
 !!! related TS6500 tests/cases/compiler/typeArgumentDefaultUsesConstraintOnCircularDefault.ts:1:37: The expected type comes from property 'value' which is declared here on type 'Test<string>'
+    
+    // Simplified repro from #28873
+    
+    class C1<T extends C1 = any> {}  // Error, circular constraint
+                       ~~
+!!! error TS2313: Type parameter 'T' has a circular constraint.

```

Comment:
Uhhhh. I don't think it's great for us to error here, TBH. That `T` should extend `C1<any>` is pretty clear.